### PR TITLE
2022.02.25 First commit for analytics strategy

### PR DESCRIPTION
**Pull request by adichourasiya**
Hi team, wrote this article to explain analytics strategy document created last year. please have a look.

### CODE REVIEW

**Comment by wpschaub** [comment]
@adichourasiya a few suggestions, whereby I am happy to make the changes for you:

- metadata date will need to be updated when you publish @wsbctechnicalblog.
- change filename to devops-analytics-strategy.md to create consistent kebab naming style.
- metadata slug should be same as filename, i.e. **devops-analytics-strategy**, for consistency.
- metadata author is incorrect. Change to **Aditya Chourasiya** and add entry to authors.
- metadata tags = learning, metrics, devops
- add bulleted lists for easier reading.
- add an explanation of what V2MOM stands for.

**Review by wsbctechnicalblog — DISMISSED** [review]
I like this article, it's succinct and to the point. Nice to have such handy, easy to find reference on our blog.

Suggestion: @adichourasiya, maybe think about writing a followup article to talk more about how you see the current situation at WSBC regarding the overall analytics strategy?

**Comment by adichourasiya** [comment]
I've made recommended changes in latest commit.

File: content/devops-analytics-strategy.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Date: 2022-01-01
+Category: Posts
+Tags: learning, metrics, devops
+Slug: devops-analytics-strategy
+Author: Aditya Chourasiya
+Summary: Objective, Method and Maturity model for business analytics across an enterprise.
+
+   As a business user, I want to make sense of all the data coming from my production system and make sure my product meets business requirements. This article summarizes different maturity model and action-sequence to achieve snow-ball effect for typical organization.
+
+![Vision](../images/vision.webp)
```
Comment (wpschaub) [inline content/devops-analytics-strategy.md]: @adichourasiya image is not showing up. Can we convert if to *.png?

File: content/devops-analytics-strategy.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+This phase is focused on developing Prescriptive Analytics capabilities, which can help advise on possible outcomes and answer: “What should we do?”
+- Skill based resource movement
+- Machine learning integration to other then business value delivery
+- Analytics Governance body
+- Business requirement generation through analytics
+- Measure how analytics is changing business
+- Strategic decision making through automation
+
+# Summary and invite
+This article was intended to be written in [V2MoM](https://www.salesforce.com/blog/how-to-create-alignment-within-your-company/) format, however due to different organizational structure and needs, it will vary a lot. Based on my experience in different government and non-government organizations, the above holds true. Please feel free to submit a pull request to suggest any changes. Thank you for reading.
```
Comment (wpschaub) [inline content/devops-analytics-strategy.md]: @adichourasiya please add two empty lines at the end.

File: content/devops-analytics-strategy.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+Title: V2MoM for DevOps Analytics
+Date: 2022-01-01
```
Comment (wpschaub) [inline content/devops-analytics-strategy.md]: @wsbctechnicalblog you will have to update this date B4 publishing.

**Review by wpschaub — CHANGES_REQUESTED** [review]
A few minor tweaks requested.

**Review by wpschaub — CHANGES_REQUESTED** [review]
@adichourasiya thank you for converting your images to png, which is more consistent with the rest of the images. That said, I finally figured out why your images are not showing up when viewing your md file. 

Please move your images from the /content/images folder to the /images folder.

cc @wsbctechnicalblog  another gremling we need to keep an eye on.

**Comment by wpschaub** [comment]
@adichourasiya I fixed the images,  made a few format and spelling update. @wsbctechnicalblog  this is good to go for February 2nd.

**Review by wpschaub — DISMISSED** [review]

**Comment by wpschaub** [comment]
@adichourasiya are we good to go with this PR? I updated the publication date to 02/25.

**Review by wpschaub — APPROVED** [review]

**Comment by wpschaub** [comment]
@adichourasiya are we good to go with this PR? I updated the publication date to 02/25.

**Pull request merged**